music21/base.py

```diff
@@ -923,7 +923,7 @@ def setOffsetBySite(self,
         0.0
         '''
         if site is not None:
-            site.setElementOffset(self, value)
+            site.coreSetElementOffset(self, value)
```

Comment:
this should definitely be the normal setElementOffset() -- so site is left intact